### ISSUE TITLE
Support for Angular 8

### ISSUE THREAD

**Issue by stevenlauwers22** [andreasonny83/angular2-cookie-law#42]
I've upgraded my application to Angular 8 and I'm getting some warnings on this package.
Could you update your package.json file to also allow angular 8, so I can get rid of these warnings

```
warning " > angular2-cookie-law@7.0.1" has incorrect peer dependency "@angular/common@^6.0.0-rc.0 || ^6.0.0 || 7.0.0-rc.0 || ^7.0.0".
warning " > angular2-cookie-law@7.0.1" has incorrect peer dependency "@angular/core@^6.0.0-rc.0 || ^6.0.0 || 7.0.0-rc.0 || ^7.0.0".
```

**Comment by evolkmann** [comment]
Hi @stevenlauwers22, since this lib seems to be discontinued unfortunately, I took the time to rewrite this library to angular 9. I also removed much of the opinionated behaviour.

Maybe this is interesting for your: https://github.com/exportarts/ngx-cookie-banner

**Comment by stevenlauwers22** [comment]
Hi @evolkmann, thanks for pointing out!

**Comment by HelainSchoonjans** [comment]
Ngx-cookie-banner unfortunately does not support Angular 8... Won't anyone fork and upgrade angular2-cookie-law?

**Comment by evolkmann** [comment]
@HelainSchoonjans You are right, it says so in the peer dependencies. I will change that in a moment and go back to v8. I did expect v9 to arrive earlier, and I am using the prerelease CLI on my machine already. Nevertheless, the software is still working with angular 8 I am pretty sure.

**Comment by evolkmann** [comment]
@HelainSchoonjans I did not fork it, because I wanted to get rid of the opinionated styling. It was easier to take only the useful parts of the logic and refactor them in the process.

**Comment by evolkmann** [comment]
> @HelainSchoonjans You are right, it says so in the peer dependencies. I will change that in a moment and go back to v8. I did expect v9 to arrive earlier, and I am using the prerelease CLI on my machine already. Nevertheless, the software is still working with angular 8 I am pretty sure.

v0.0.3 has now been released: https://www.npmjs.com/package/ngx-cookie-banner

**Comment by andreasonny83** [comment]
I'm sorry but I'm not actively maintaining the package any more.
I will be happy to accept a PR to add compatibility from major Angular versions if you can help.

Thanks
